automotive/src/main/java/au/com/shiftyjelly/pocketcasts/AutomotiveApplication.kt

```diff
@@ -120,8 +118,4 @@ class AutomotiveApplication : Application(), Configuration.Provider {
             Timber.plant(TimberDebugTree())
         }
     }
-
-    private fun setupAnalytics() {
-        AnalyticsTracker.init(settings)
-    }
```

Comment:
I noticed that Automotive app unlike Phone and Wear apps does not refresh trackers metadata here. I'm not sure if it is a bug but I kept previous behavior.